lms/djangoapps/program_enrollments/management/commands/link_program_enrollments.py

```diff
@@ -150,11 +159,12 @@ def link_program_enrollment(self, program_enrollment, user):
         for program_course_enrollment in program_enrollment.program_course_enrollments.all():
             try:
                 program_course_enrollment.enroll(user)
-            except CourseEnrollmentException:
+            except CourseEnrollmentException as e:
                 logger.warning(COURSE_ENROLLMENT_ERR_TPL.format(
                     user=user.username,
                     course=program_course_enrollment.course_key
                 ))
+                raise e
```

Comment:
Style tip: Bare `raise` will re-raise the exception for the block you're in, so you can leave out `e`

Reply:
good to know!